### PR TITLE
Adds nvhpc 23.11

### PR DESCRIPTION
**Pull request by Julius-Plehn**

### CODE REVIEW

**Comment by samcmill** [comment]
Duplicate of https://github.com/spack/spack/pull/41125

**Comment by alalazo** [comment]
@samcmill I just merged #41125, but noticed that 2 shas out of 3 are different in this PR. I am checking right now which one is correct.

File: var/spack/repos/builtin/packages/nvhpc/package.py [hunk 1]
```diff
@@ -21,6 +21,20 @@
...
 _versions = {
+    "23.11": {
+        "Linux-aarch64": (
+            "03397699b38d06749fb38ee285f2a69d08f5eec71ea3a89968ba9b5a81933d5d",
+            "https://developer.download.nvidia.com/hpc-sdk/23.11/nvhpc_2023_2311_Linux_aarch64_cuda_multi.tar.gz",
+        ),
+        "Linux-ppc64le": (
+            "1ea690be40f2af4b0a5b46e30d1b8603f7ef80f74635f2908b21ce3f3da1e9a1",
+            "https://developer.download.nvidia.com/hpc-sdk/23.11/nvhpc_2023_2311_Linux_ppc64le_cuda_multi.tar.gz",
+        ),
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/nvhpc/package.py:32]: These two sha256 seems not to be correct.

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by alalazo** [comment]
@Julius-Plehn Closing as duplicate, `nvhpc@=23.11`  is in `develop`

**Pull request closed without merging**